website/src/views/covid.ts

```diff
@@ -325,3 +325,9 @@ class CovidSequencingEffortsStateHandler
         return `${this.pathname}?${search}`;
     }
 }
+
+export class CovidCompareVariantsView extends GenericCompareVariantsView<CovidConstants> {
+    constructor(organismsConfig: OrganismsConfig) {
+        super(new CovidConstants(organismsConfig));
+    }
+}
```

Comment:
I'm not sure whether we directly want to support `variantQuery`s  here. But let's leave it as it is until we really need it. (I think that's the reason why we have separate logic in the other views for covid)

Reply:
We have the variant query for covid, because we use it for the collections on the analyze-single-variant-view. But we have no input element to get the variant query otherwise. I think we leave it like this and only add it, when we have a variantQuery filter element on the page.